src/ansys/dpf/post/dpf_solution.py

```diff
@@ -33,25 +36,25 @@ class DpfSolution:
     """Main class of post result API."""
 
     def __init__(self, data_sources, model):
-        """Initialization of the solution using data_sources
-        and dpf.core.Model object."""
+        """Initialize the solution using data_sources and dpf.core.Model object."""
         self._data_sources = data_sources
         self._model = model
 
     @property
     def mesh(self):
-        """Mesh representation of the model.
-        Based on the dpf.core.MeshedRegion class."""
+        """Return the Mesh representation of the model.
```

Comment:
```suggestion
        """Mesh representation of the model.
```

Reply:
@akaszynski You taught me that @property decorators should have noun (not verb) descriptions

Reply:
`pydocstyle` demands that all methods start with a verb, even if that method is a `property`. However, we can ignore this with `--ignore-decorators`.

Thanks for having me research this. It's been bothering me for a while.